packages/terminal/src/node/shell-terminal-server.ts

```diff
@@ -90,10 +96,26 @@ export class ShellTerminalServer extends BaseTerminalServer implements IShellTer
         if (processId) {
             // if shell has at least one child process, assume that shell is busy
             if (isWindows) {
-                return this.spawnAsPromised('wmic', ['process', 'get', 'ParentProcessId']).then(stdout => {
-                    const pids = stdout.split('\r\n');
-                    return pids.some(p => parseInt(p) === processId);
-                }, error => true);
+                return new Promise(resolve => {
+                    exec(
+                        'powershell -Command "Get-CimInstance Win32_Process | Select-Object ProcessId, ParentProcessId | ConvertTo-Json"',
+                        (error, stdout, _stderr) => {
+                            if (error) {
+                                console.error(`Failed to get Windows process list: ${error}`);
```

Comment:
Minor: Please use the logger
```suggestion
                                this.logger.error(`Failed to get Windows process list: ${error}`);
```